.github/workflows/test.yml

```diff
@@ -227,7 +227,7 @@ jobs:
       - name: Install dependencies (Bash)
         shell: bash -l {0}
         run: |
-          pip install --requirement requirements.txt
+          pip install --requirement requirements.txt --user
```

Comment:
--user breaks virtual environment. With --user pip will install packages globally that will create a massive mess (speaking from experience).